buildMac.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+set -e
+
+if [ "$GITHUB_ACTIONS" == "true" ]
+    then
+        testLogger="GitHubActions;report-warnings=false"
+    else
+        testLogger="console"
+fi
+
+dotnet test $solutionFilterFile -c Release -l $testLogger \
```

Comment:
The idea here is to delete the `buildLinux` and `buildMac` and have a single script